packages/airnode-node/src/handlers/start-coordinator.test.ts

```diff
@@ -172,7 +172,12 @@ describe('startCoordinator', () => {
     await startCoordinator(config, coordinatorId);
 
     // cached requests should not trigger an API call
-    expect(callApisSpy).toHaveBeenCalledWith([], expect.objectContaining({ deploymentId: 'localdd59d6d0' }));
+    expect(callApisSpy).toHaveBeenCalledWith(
+      [],
+      expect.objectContaining({
+        deploymentId: expect.stringMatching(new RegExp(`local[0-9a-z]{${DEPLOYMENT_ID_LENGTH}}`)),
```

Comment:
Since the regular expression is always the same we can extract it as a constant and use that instead.
Also, it can be made more strict using `[0-9a-f]` (only hexadecimal) instead.

Reply:
A constant across all tests? Without the `local` prefix? I may not be understanding how you envision this. 

Also `a-f`, right haha.

Reply:
I've updated it to `a-f` and to have a `const` regex variable per test file. I don't think it makes sense to have it as a "global" constant because the cloud provider type prefix of `local` applies to these tests and wouldn't apply to `aws` or `gcp` deployment ID regexes

Reply:
I mean it as a global test constant since all the tests here use only the `local` prefix. But this is fine as well.